### PR TITLE
feat : added carts component

### PR DESCRIPTION
**Pull request by Priya-Patel31**
It consists of :
* Products in the carts
* Total price summary

### CODE REVIEW

File: carts/carts.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+                    Remove from cart
+                  </button>
+                  <button class="button secondary-button-light">
+                    Add to wishlist
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-40-pc boder-line flex-col">
```
Comment (t007rushi) [inline carts/carts.html]: add some padding to billing card

File: carts/carts.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+.width-800 {
+    max-width: 950px;
+    display: flex;
+    margin: 0 auto;
+    flex-wrap: wrap;
+    gap: 1.5rem;
+  }
+  .cart-heading {
```
Comment (t007rushi) [inline carts/carts.css]: add some left margin

File: carts/carts.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    text-align: center;
+  }
+  .col-60-pc {
```
Comment (t007rushi) [inline carts/carts.css:36]: here also margin-left required as it touching viewport border

File: carts/carts.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+            <p class="font-bold mb-1 text-xs">COUPONS</p>
+            <button class="button secondary-button-light">
+              <span><i class="fas fa-tag mr-1 icon-black text-xs"></i></span>Apply
```
Comment (t007rushi) [inline carts/carts.html:133]: apply coupon if match the place order  btn width will look good or use fit-content make it small as it isn't that important while checkout

**Review by t007rushi — CHANGES_REQUESTED** [review]
little margin & padding will improve UI else clean as always

File: carts/carts.html (hunk 1 above)
Comment (Priya-Patel31) [reply carts/carts.html]: CSS was not getting applied that is why this issue is happening !!

File: carts/carts.html (hunk 4 above)
Comment (Priya-Patel31) [reply carts/carts.html:133]: I have made changes in it !!

File: carts/carts.css (hunk 2 above)
Comment (Priya-Patel31) [reply carts/carts.css]: same as above

File: carts/carts.css (hunk 3 above)
Comment (Priya-Patel31) [reply carts/carts.css:36]: same as above

File: carts/carts.css [hunk 5]
```diff
@@ -0,0 +1,54 @@
+.width-800 {
+  max-width: 950px;
```
Comment (Kothalkarpriya) [inline carts/carts.css]: use rem for more that 10px

**Review by Kothalkarpriya — COMMENTED** [review]
Look into my comments, after that you can merge

File: carts/carts.css [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+.cart-heading {
+  text-align: center;
+}
+.col-60-pc {
+  width: calc(60% - 2rem);
+}
+.col-40-pc {
+  width: 40%;
+  box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+  padding: 0.5rem 1rem;
```
Comment (Ankur9669) [inline carts/carts.css:17]: You can use CSS variiable for box-shadow

File: carts/carts.html [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+                    <i class="fas fa-plus"></i>
+                  </button>
+                  <p class="text-xs">3</p>
+                  <button class="floating-action-btn">
+                    <i class="fas fa-minus"></i>
+                  </button>
+                </div>
+                <span class="discount-price font-semibold">Rs.1500 </span>
+                <span class="original-price"> Rs.2200</span>
+                <small class="discount-rate">(15% OFF)</small>
```
Comment (Ankur9669) [inline carts/carts.html]: span is not a semantic element, so you can replace span with p

**Review by Ankur9669 — APPROVED** [review]
Code and UI looks great!!!. I have given some suggestions you can work on them

File: carts/carts.css (hunk 5 above)
Comment (Priya-Patel31) [reply carts/carts.css]: changed it to rems !!

File: carts/carts.html (hunk 7 above)
Comment (Priya-Patel31) [reply carts/carts.html]: changed it p tag !!

File: carts/carts.css (hunk 6 above)
Comment (Priya-Patel31) [reply carts/carts.css:17]: changed it with variable name

**Comment by Priya-Patel31** [comment]
> Code and UI looks great!!!. I have given some suggestions you can work on them

**Comment by Priya-Patel31** [comment]
> > Code and UI looks great!!!. I have given some suggestions you can work on them

Worked on it and made the required changes !!

**Review by t007rushi — APPROVED** [review]

**Pull request merged**